### PR TITLE
Create the Enumerable 

### PR DESCRIPTION
**Pull request by AbbyNyakara**
In this PR, we created the enumerables for any?, all? and filter.
 The project requirements included: 

Check offical documentation about the following 3 methods in [Enumerable](https://ruby-doc.org/core-3.0.0/Enumerable.html). Make sure that you understand what they are doing.
[description of #all? method](https://ruby-doc.org/core-3.0.0/Enumerable.html#method-i-all-3F)
[description of #any? method](https://ruby-doc.org/core-3.0.0/Enumerable.html#method-i-any-3F)
[description of #filter method](https://ruby-doc.org/core-3.0.0/Enumerable.html#method-i-filter)
Create a class MyList that has an instance variable @list.
In MyList implement a method #each that yields successive members of @list and uses the MyEnumerable module.
Create a module MyEnumerable that implements the following methods (they should have the same funcionality as methods in [Enumerable](https://ruby-doc.org/core-3.0.0/Enumerable.html)):
#all?
#any?
#filter
Each class and module should has a separate .rb file.

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require './my_enumerable'
```
Comment (danmainah) [inline my_list.rb]: - [x] Kindy use require relative to import your `my_enumerable`, this is ruby convection to use require relative for files in your directory and require for external files like gems.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️

- [ ] Kindly check how you import your enumerable am getting this error when running the tests:
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: my_list.rb (hunk 1 above)
Comment (AbbyNyakara) [reply my_list.rb]: Oh. Nice. Didnt know this

**Review by sumairq — APPROVED** [review]
### Hi Team , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**